### PR TITLE
add an extra note to clusterapi readme about gpus

### PR DESCRIPTION
**Pull request by elmiko**
#### What type of PR is this?

/kind cleanup
/kind documentation

#### What this PR does / why we need it:

this change adds a little more detail to ensure that users understand how to use the GPU label feature.

#### Which issue(s) this PR fixes:

Fixes #

#### Does this PR introduce a user-facing change?

```release-note

```

#### Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.:

```docs

```

### CODE REVIEW

File: cluster-autoscaler/cloudprovider/clusterapi/README.md [hunk 1]
```diff
@@ -361,6 +361,11 @@ CAPI cloudprovider, the label format is as follows:
 
 `<gpu-type>` is arbitrary.
 
+It is important to note that if you are using the `--gpu-total` flag to limit the number
+of GPU resources in your cluster that the `<gpu-type>` value must match
```
Comment (jackfrancis) [inline cluster-autoscaler/cloudprovider/clusterapi/README.md:365]: Can we provide a concrete known-working example that demonstrates CLI flag <--> node label affinity? That would help solidify the requirement concepts here, I think.

Comment (elmiko) [reply cluster-autoscaler/cloudprovider/clusterapi/README.md:365]: absolutely

**Comment by elmiko** [comment]
updated with example

**Review by jackfrancis — APPROVED** [review]
lgtm

**Pull request merged**